pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -243,7 +243,7 @@ func prioritize(context context.Context, schedulerExtenderArgs schedulerapi.Exte
 		go getAzVolumeAttachments(context, volumesChan)
 
 		// create a lookup map of all the volumes the pod needs
-		volumesPodNeeds, _ := getDisksRequestedByPod(ns, requestedVolumes)
+		volumesPodNeeds, _ := getDisksRequestedByPod(ns, requestedVolumes, v1.ReadOnlyMany, v1.ReadWriteMany)
```

Comment:
Sorry for being neat picky here, I know I only asked for this change, but looking at it again I think this is still a really easy mistake to make in the future. To keep the change in this PR minimal, maybe we should rename the function to `getDisksForPodExcludeAccessMode`